build/scripts/build.ps1

```diff
@@ -1,6 +1,6 @@
 #
 # This script controls the Roslyn build process. This encompasess everything from build, testing to
-# publishing of NuGet packages. The intent is to structure it to allow for a simple flow of logic 
+# publishing of NuGet packages. The intent is to structure it to allow for a simple flow of logic
```

Comment:
Lots of unrelated changes in the file will make a mess of git blame and now I have to go find the relevant change(s). ☹️

Reply:
I can add back all the whitespace